### ISSUE TITLE
[SGX] PAL always returns 0 regardless of payload's exit code

### ISSUE THREAD

**Issue by omeg** [oscarlab/graphene#205]
```
~/graphene/Runtime/pal-Linux-SGX test.manifest.sgx; echo "ret=$?"
...
[     1] ---- shim_exit_group (returning 1)
[     1] now kill other threads in the process
[     1] walk_thread_list(callback=0xb8de7a8)
[     1] now exit the process
[     1] ipc broadcast: IPC_CLD_EXIT(1, 1, -1)
[     1] found port 0xb8280c0 (handle 0xbd60f50) for process 0 (type 0002)
[     1] found port 0xb828048 (handle 0xbd606d0) for process 0 (type 0001)
[     1] parent not here, need to tell another process
[     1] ipc broadcast: IPC_CLD_EXIT(1, 1, -1)
[     1] found port 0xb8280c0 (handle 0xbd60f50) for process 0 (type 0002)
[     1] found port 0xb828048 (handle 0xbd606d0) for process 0 (type 0001)
[     1] this is the only thread
[     1] exiting ipc helper
[P22466] ipc helper thread terminated
[     1] deleting port 0xb8280c0 (handle 0xbd60f50) for process 0
[     1] deleting port 0xb828048 (handle 0xbd606d0) for process 0
[     1] process 22466 successfully terminated
ret=0
```

`test` is just doing `exit(1)`.

**Comment by donporter** [comment]
Thanks for the report @omeg .  PR #216 should fix this issue.

**Comment by mkow** [comment]
This was fixed by #216.
